annotated-projects/PXD025168/sdrf-plants.tsv

```diff
@@ -0,0 +1,127 @@
+source name	characteristics[organism]	characteristics[organism part]	characteristics[cell type]	characteristics[disease]	characteristics[biological replicate]	technology type	assay name	comment[technical replicate]	comment[data file]	comment[fraction identifier]	comment[label]	comment[instrument]	comment[cleavage agent details]	comment[modification parameters ]	comment[modification parameters ]	comment[modification parameters ]

```

Comment:
We recommend to use a factor value for each SDRF, a varaible or set of variables from the samples properties or data properties to study in the experiment.